src/backend/gl/src/lib.rs

```diff
@@ -127,6 +127,9 @@ impl ResourceView {
             owned: true,
         }
     }
+    pub fn raw_texture(&self) -> &Texture {
```

Comment:
since GL knows this is integer, we might just return it by copy. Also, for clarity we can rename this to `gl_texture`